deploy

```diff
@@ -4,6 +4,8 @@
 
 REPO=$TRAVIS_REPO_SLUG
 
+[[ -z $ITB_BRANCH_REGEXP ]] && ITB_BRANCH_REGEXP="itb\..*"
```

Comment:
A second (minor) comment about this is i usually recommend

`[[ $SOMEVAR ]] || SOMEVAR="somedefault"`

over

`[[ -z $SOMEVAR ]] && SOMEVAR="somedefault"`

because the first expression always returns success, but it's a bit of a style choice and either way does actually work.